src/App.tsx

```diff
@@ -1,12 +1,73 @@
 import React from 'react';
 import './App.scss';
+import '../node_modules/bootstrap/dist/css/bootstrap.min.css';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import { getAll, get5First, getRedGoods } from './api/goods';
+import GoodsList from './components/GoodsList';
 
-const App: React.FC = () => (
-  <h1>Dynamic list of Goods</h1>
-);
+type State = {
+  goods: Good[];
+};
+
+class App extends React.Component<{}, State> {
+  state: State = {
+    goods: [],
+  };
+
+  setAllGoods = async () => {
+    const goods = await getAll();
+
+    this.setState({ goods });
+  };
+
+  set5First = async () => {
+    const goods = await get5First();
+
+    this.setState({ goods });
+  };
+
+  setRedGoods = async () => {
+    const goods = await getRedGoods();
+
+    this.setState({ goods });
+  };
```

Comment:
you can create 1 common function for getting goods, and just pass the needed function on the argument  when press button